### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by mj-e-boc**
-Add contact section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -142,6 +142,18 @@ <h2 class="topic">Multi-Post Stories</h2>
...
+    <section class="contact_section">
+        <h2 class="contactme">CONTACT US</h2>
+        <p class="cpara">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.
+        </p>
+        <form action="https://formspree.io/f/mdobrazo" method="POST">
+            <input class="textinput" type="text" name="name" placeholder="Text Input" maxlength="30" required="required" />
+            <input class="textinput" type="email" placeholder="youremail@email.com" required="required" />
+            <textarea class="textarea" cols="30" rows="10" placeholder="Enter text here.." maxlength="500" required="required"></textarea>
+            <button class="subbutton button" type="submit">Submit</button>
```
Comment (tufoinnkuo10) [inline index.html:154]: - [ ] Great job with the contact section. kindly add the last social icons under your contact so u can be reached easily from any part of the page.
- your version
Screenshot (206)
figma version
Screenshot (205)_LI

File: index.html [hunk 2]
```diff
@@ -142,6 +142,18 @@ <h2 class="topic">Multi-Post Stories</h2>
...
     </div>
+    <section class="contact_section">
+        <h2 class="contactme">CONTACT US</h2>
+        <p class="cpara">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.
+        </p>
+        <form action="https://formspree.io/f/mdobrazo" method="POST">
+            <input class="textinput" type="text" name="name" placeholder="Text Input" maxlength="30" required="required" />
+            <input class="textinput" type="email" placeholder="youremail@email.com" required="required" />
+            <textarea class="textarea" cols="30" rows="10" placeholder="Enter text here.." maxlength="500" required="required"></textarea>
```
Comment (tufoinnkuo10) [inline index.html:153]: - [ ] please add a little bit of left and right margin to the input fields so they look more like the figma version.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @mj-e-boc 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ figma design followed
✅ form submits successfully

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by topeogunleye — APPROVED** [review]
### Hi @mj-e-boc, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
 
> ## [Optional] suggestions:
>- [ ] Please kindly add a more descriptive PR title and descriptive.
> See a demo below:

https://user-images.githubusercontent.com/52934128/176308229-e31b2c8d-110b-4823-a2a8-31c1b7d7c73d.mp4

>- [ ] I kindly suggest that you continue to make use of imperative mood in the subject line for your commit messages.
:x: Wrong
image

> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>
>- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**